lib/id3c/cli/command/__init__.py

```diff
@@ -125,7 +126,9 @@ def pickled_cache(filename: str = None) -> TTLCache:
         LOG.info(f"Loading cache from «{filename}»")
         try:
             with open(filename, "rb") as file:
+                lock_file(file)
```

Comment:
I can't comment on the implementation of `lock_file()` yet, but I think the usage here isn't doing what you expect because it's not being called as a context manager. It should be called as:

```python
with lock_file(file):
    …
```